src/camera.cpp

```diff
@@ -288,14 +289,15 @@ Camera::Device::Device()
   m_cap(0),
   m_image(),
   m_resolution(HIGH_RES),
-  m_model(/*WHITE_2016*/ BLACK_2017)
+  m_model(/*WHITE_2016*/ /*BLACK_2017*/ TELLO)
 {
   Config *config = Config::load(Camera::ConfigPath::defaultConfigPath());
+  ::printf("camera open\n");fflush(NULL);
```

Comment:
Use iostreams. `fflush()` doesn't make sense regardless, as stdout flushes on newline

Reply:
Actually fflush is not a part of iostream namespace. Interestingly, I had to add "using namespace std" before it recognize iostream.

Also, I have to disagree about stdout flushing upon a newline. My experience here and on other projects (even with cout), if you do not put in fflush(NULL) and the program crashes, that line will not be outputted. I don't suggest it be used everywhere as there is a performance impact, but for error messages I find that they are a necessity if you want to get a idea of an error that may have caused a crase.

Reply:
At least for iostreams the C++ specification states that `std::endl` (`'\n'`) [flushes the buffer](https://en.cppreference.com/w/cpp/io/manip/endl). I recognize the debugging concern, but the `fflush()` is by definition not necessary.

Reply:
Did some digging and found that  std::endl is supposed to be equivent to '\n' + flush. '\n' does not cause a flush. With printf you have to use fflush. of course, cout only needs endl

Reply:
I believe that \n in the format string of printf will cause a flush. stdout is line buffered, and this is a property of the file descriptor itself.

Reply:
I have to disagree.  In my almost 40 years of dealing with C '\n' has never forced a flush. And my experience with C++ has been the same. And my work in this project has changed that experice as well You have to tell stdout to flush the buffer! Let's just agree to disagree as you may have experience with different runtimes and compilers.